### PR TITLE
[READY]Plant mutation chances at 80+ instability is now a sliding scale.

### PR DESCRIPTION
**Pull request by KathyRyals**
## About The Pull Request

After discussing it with @ArcaneMusic, we agreed that the current chance to get a random trait / chemicals at 80 + instab is wayyyy too low. The chance is now a sliding scale with a 5% chance at 80 instability, going up to 25% at 100 instability.

## Why It's Good For The Game

More opportunities to get cool chemicals and trait as Botanist, without having to rely on cheesing Strange Seeds, or planting a ton of high instability plant to probably only get a couple rolls anyway.

## Changelog
:cl: Kathy Ryals and ArcaneMusic
tweak: Plant mutation chances at 80+ instability is now a sliding scale.
/:cl:

### CODE REVIEW

File: code/modules/hydroponics/hydroponics.dm [hunk 1]
```diff
@@ -256,7 +256,8 @@
 
 //This is where stability mutations exist now.
 			if(myseed.instability >= 80)
-				mutate(0, 0, 0, 0, 0, 0, 0, 5, 0) //Exceedingly low odds of gaining a trait.
+				var/mutation_chance = 5 + (100 - myseed.instability)
```
Comment (LemonInTheDark) [inline code/modules/hydroponics/hydroponics.dm:259]: doesn't this lower the mutation chance the higher the instab is?

**Comment by ATH1909** [comment]
do note that random mutations from instability CAN affect your instability score, so don't expect for instability to remain at 100 for a lengthy period of time

still, this is a good idea, I think

**Comment by Krysonism** [comment]
As someone who has made plenty of super death plants with the old system, this is a huge buff.

File: code/modules/hydroponics/hydroponics.dm (hunk 1 above)
Comment (KathyRyals) [reply code/modules/hydroponics/hydroponics.dm:259]: Check the surrounding code again, we are in a special bit that is only called when instab > 80 already

Comment (LemonInTheDark) [reply code/modules/hydroponics/hydroponics.dm:259]: So if instab > 80
chance = 5 + (100 - instab)
plugging in some numbers,
f(x) = 5 + (100 - x)
f(81) = 5 + 19 = 24
f(99) = 5 + 1 = 6
This doesn't sound like the behavior you want

**Comment by Angustmeta** [comment]
2% felt low but 25% might be an overbuff.

File: code/modules/hydroponics/hydroponics.dm (hunk 1 above)
Comment (KathyRyals) [reply code/modules/hydroponics/hydroponics.dm:259]: wooooooooppppppppppppsssssssssssssssssssssss

**Comment by KathyRyals** [comment]
The high chance is balanced by the fact that you need to cull all the unwanted traits and chemicals out of the plant after you're done mutating it.

**Comment by KathyRyals** [comment]
@LemonInTheDark Big thanks, formula should be feexed now

**Comment by LemonInTheDark** [comment]
blessed

**Comment by kriskog** [comment]
Give it a go, tune it down if its too much.

**Pull request merged**